samza-core/src/main/java/org/apache/samza/config/ZkConfig.java

```diff
@@ -28,8 +28,10 @@
 
   public static final int DEFAULT_CONNECTION_TIMEOUT_MS = 60000;
   public static final int DEFAULT_SESSION_TIMEOUT_MS = 30000;
-  public static final int DEFAULT_CONSENSUS_TIMEOUT_MS = 40000;
```

Comment:
It was not being used. After rebasing with master (I think you had a few commits), it was not being used. So, I removed it. Let me know if I merged incorrectly. There were some conflicts during rebase.